config/functions/functions_usage_summary.md

```diff
@@ -0,0 +1,548 @@
+---
+title: Functions Template Syntax Summary
+---
+
+Here’s a summary list of all of the "Usage: " comment sections from the header files for  
+functions as of ProffieOS 7.13.  
```

Comment:
This is actually incorrect, some of these functions do not exist in 7.13.

Reply:
removed bulletcount, pixelate, and readpin for now. should be same as 7.13 now?